src/Api.Rest/Common/Client/RestClientBase.cs

```diff
@@ -253,19 +256,39 @@ public Task Request( [NotNull] Func<HttpRequestMessage> requestCreationHandler,
 			return PerformRequestAsync<object>( requestCreationHandler, false, null, true, cancellationToken );
 		}
 
+		public Task Request( [NotNull] Func<IObjectSerializer, HttpRequestMessage> requestCreationHandler, CancellationToken cancellationToken )
+		{
+			return PerformRequestAsync<object>( () => requestCreationHandler(_Serializer), false, null, true, cancellationToken );

```

Comment:
Code cleanup of the file with ReSharper should fix missing spaces in all brackets.

Reply:
Fixed